docs/install.md

```diff
@@ -49,7 +49,7 @@
 ## Installing latest release
 
 ```sh
-$ kubectl apply -f https://storage.googleapis.com/tekton-releases/results/previous/v0.3.0/release.yaml
+$ kubectl apply -f https://storage.googleapis.com/tekton-releases/results/previous/v0.5.0/release.yaml
```

Comment:
Now we have link to latest: https://storage.googleapis.com/tekton-releases/results/latest/release.yaml
Use it and we won't need to update this anymore.